video/profiles.go

```diff
@@ -89,22 +89,23 @@ func GetPlaybackProfiles(iv InputVideo) ([]EncodedProfile, error) {
 	if err != nil {
 		return nil, fmt.Errorf("no video track found in input video: %w", err)
 	}
+	videoBitrate := video.Bitrate
+	if videoBitrate > MaxVideoBitrate {
+		videoBitrate = MaxVideoBitrate
+	}
 	profiles := make([]EncodedProfile, 0, len(DefaultTranscodeProfiles)+1)
 	for _, profile := range DefaultTranscodeProfiles {
 		// transcoding job will adjust the width to match aspect ratio. no need to
 		// check it here.
-		lowerQualityThanSrc := profile.Height < video.Height && profile.Bitrate < video.Bitrate
+		lowerQualityThanSrc := profile.Height < video.Height
 		if lowerQualityThanSrc {
+			profile.Bitrate = (profile.Width * profile.Height * videoBitrate) / (video.Width * video.Height)
```

Comment:
nit: I wonder if we need to make this more overflow-safe. There are still ~12 bits of buffer on our current max values (4k @ 288mbps), but these consts can change and this calculation become error prone.

Maybe multiply `videoBitrate` by `dstWidth/srcWidth` then `dstHeight/srcHeight` as floats then come back to an `int64`?

Reply:
good point 👍